### PR TITLE
Add trailing slash to the graphql URLs

### PR DESCRIPTION
**Pull request by mathjazz**
Both of these URLs for:

https://mozilla-pontoon-staging.herokuapp.com/graphql?query=%7Bprojects%7Bname,missingStrings,approvedStrings%7D%7D

https://mozilla-pontoon-staging.herokuapp.com/graphql/?query=%7Bprojects%7Bname,missingStrings,approvedStrings%7D%7D

### CODE REVIEW

File: pontoon/api/tests/test_urls.py [hunk 1]
```diff
@@ -47,22 +47,22 @@ def test_graphql_prod_get(settings, projects_query, client):
 def test_graphql_prod_post(settings, projects_query, client):
     settings.DEV = False
 
-    response = client.post("/graphql", projects_query, HTTP_ACCEPT="appication/json")
+    response = client.post("/graphql/", projects_query, HTTP_ACCEPT="appication/json")
```
Comment (flodolo) [inline pontoon/api/tests/test_urls.py:50]:
```suggestion
    response = client.post("/graphql/", projects_query, HTTP_ACCEPT="application/json")
```

I'm assuming the typo is not wanted here?

**Review by flodolo — CHANGES_REQUESTED** [review]
1) Do we want to update the links in the [documentation](https://github.com/mozilla/pontoon/blob/main/pontoon/api/README.md)? 

BTW, the `curl` example doesn't work on macOS+zsh without quotes 

```
curl --globoff "http://localhost:8000/graphql?query={projects{name}}"
```

2) As I mentioned in the other PR, this works in the browser on DEV where the URL is redirected (e.g. `graphql?query={projects{name}}` ⇾ `graphql/#query=%7B%0A%20%20projects%20%7B%0A%20%20%20%20name%0A%20%20%7D%0A%7D`. 

But it doesn't work when I try with curl (even on DEV, no need to switch to PROD): I get an empty answer, while if I manually add the `/` I get an answer.

```
$ curl --globoff -w "\nStatus code: %{http_code} \n" "http://localhost:8000/graphql?query={projects{name}}"


Status code: 301
$ curl --globoff -w "\nStatus code: %{http_code} \n" "http://localhost:8000/graphql/?query={projects{name}}"

{"data":{"projects":[{"name":"Terminology"}]}}
Status code: 200
```

**Comment by mathjazz** [comment]
Explicitly added support for the URLs with trailing slashes.

I've updated the docs, to make the URLs consistent with the TM and Terminology download URLs.

**Comment by mathjazz** [comment]
> BTW, the `curl` example doesn't work on macOS+zsh without quotes

I couldn't reproduce that.

File: pontoon/api/README.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ the query must be escaped.
 An example GET requests may look like this:
 
 ```bash
-$ curl --globoff https://example.com/graphql?query={projects{name}}
+$ curl --globoff https://example.com/graphql/?query={projects{name}}
```
Comment (flodolo) [inline pontoon/api/README.md:15]:
```suggestion
$ curl --globoff "https://example.com/graphql/?query={projects{name}}"
```

File: pontoon/api/README.md [hunk 3]
```diff
@@ -33,23 +33,23 @@ a production one, returning JSON responses.
 The following query in the CLI will return a JSON response:
 
 ```bash
-$ curl --globoff http://localhost:8000/graphql?query={projects{name}}
+$ curl --globoff http://localhost:8000/graphql/?query={projects{name}}
```
Comment (flodolo) [inline pontoon/api/README.md:36]:
```suggestion
$ curl --globoff "http://localhost:8000/graphql/?query={projects{name}}"
```

**Review by flodolo — APPROVED** [review]
Looks good 👍🏼 

Can we quote the curl addresses with `{}` though? This works in both bash and zsh

**Pull request merged**